### ISSUE TITLE
The loss dose not decrease

### ISSUE THREAD

**Issue by abc8350712** [balancap/SSD-Tensorflow#76]
I use VOC2007 datasets to train SSD.The iteration has come to 10000, but the loss does not decrease.
Who can help me?

**Comment by congjianting** [comment]
@abc8350712 i face the same thing, did you find why?

**Comment by chenjjjjuu** [comment]
mee to
